packages/paste-website/src/components/customization-landing-page/ReadyToGetStarted.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import {Heading} from '@twilio-paste/heading';
+import {Column, Grid} from '@twilio-paste/grid';
+import {Card} from '@twilio-paste/card';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Anchor} from '@twilio-paste/anchor';
+
+import {LandingPageSection, LandingPageSectionContent} from './LandingPageLayoutUtils';
+
+export const ReadyToGetStarted: React.FC = () => {
+  return (
+    <LandingPageSection>
+      <LandingPageSectionContent variant="default">
+        <Heading as="h2" variant="heading30">
+          Ready To Get Started?
+        </Heading>
+        <Grid
+          gutter={['space0', 'space40', 'space40', 'space40']}
+          vertical={[true, true, true, false]}
+          equalColumnHeights
+        >
+          <Column span={[12, 12, 12, 6]}>
+            <Grid gutter={['space40', 'space40', 'space40']} vertical={[true, false, false]} equalColumnHeights>
+              <Column span={[12, 6, 6]}>
+                <Card>
+                  <Heading as="h3" variant="heading30">
+                    Customization provider docs
+                  </Heading>
+                  <Paragraph>
+                    Learn how to install the Customization Provider to your Twilio and Paste-powered app.
+                  </Paragraph>
+                  <Anchor href="">Read the Provider docs</Anchor>
```

Comment:
Is `href` intentionally an empty string? If this is something that will get fixed later, might it be worth adding a comment and "todo" note here so it's not missed?

Reply:
I'll add em in, I just forgot them